blob/fileblob/fileblob.go

```diff
@@ -107,6 +107,7 @@ func (b *bucket) Attributes(ctx context.Context, key string) (driver.Attributes,
 	}
 	return driver.Attributes{
 		ContentType: xa.ContentType,
+		Metadata:    xa.Metadata,
```

Comment:
This is missing a pass to lowercase all the keys.

Reply:
Hmm, the point is to do all of the lowercasing in the concrete type, so that the individual driver implementations don't have to do it. So, there's a ToLower in blob.Bucket.Attributes for reading, and one in blob.NewWriter for writing.